### PR TITLE
Update boilerplates.md

### PR DESCRIPTION
**Pull request by rokoroku**
Hi, I made a new boilerplate with mobx.
Hope this is helpful.

### CODE REVIEW

File: docs/faq/boilerplates.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * [generator-mobx-react](https://www.npmjs.com/package/generator-mobx-react) Yeoman MobX, React, Webpack, Babel / Typescript project generator
 * [mobx-starter](https://github.com/nightwolfz/mobx-starter): Starting base for an mobx react project with optional isomorphism. MongoDB auth & sessions, hot reload, react-router
 * [React, React-Router 4, MobX and Webpack 2-boilerplate with async routes](https://github.com/mhaagens/react-mobx-react-router4-boilerplate)
+* [react-mobx-typescript-boilerplate](https://github.com/rokoroku/react-mobx-typescript-boilerplate) A minimum frontend boilerplate including TodoMVC example
```
Comment (benjamingr) [inline docs/faq/boilerplates.md]: Probably mention Webpack 2 here since it's a difference from the standard official boilerplate.

Comment (rokoroku) [inline docs/faq/boilerplates.md]: Hmm, OK, I'll change the description.

Comment (rokoroku) [inline docs/faq/boilerplates.md]: Hmm, OK, I'll change the description.

**Comment by benjamingr** [comment]
LGTM

**Pull request merged**